DOC-3356: New feature documentation for `tinymceai`. (#3976)
tinymce/tinymce-docs: modules/ROOT/pages/tinymceai-introduction.adoc

= TinyMCE AI Introduction
:navtitle: Introduction
:pluginname: TinyMCE AI
:plugincode: tinymceai
:description: Introduction to TinyMCE AI features and capabilities
:description_short: TinyMCE AI introduction
:keywords: AI, introduction, tinymceai
:page-role: -toc

> By integrating AI writing assistance directly into the editing experience, {pluginname} empowers authors with real-time AI writing support, speeds up content creation, and enhances editorial workflows across a wide range of use cases, from productivity boosts and proof-reading to content quality and consistency.

Reviewer: this talks only about the plugin, not also the API, so suggesting:
```suggestion
Integrate real-time AI writing assistance into your application with TinyMCE AI, with ready-to-go editor features available via the TinyMCE plugin, and custom functionality available via the API -- even outside the editor. Speed up content creation and enhance editorial workflows across a wide range of use cases, from productivity boosts and proof-reading to content quality and consistency. All with your choice of LLM -- including your own.
```
bit too marketing-voice? I think it gets across all the key points though.

[[demo]]
== Demo

liveDemo::tinymceai[]

[[what-is-tinymce-ai]]
== What is {pluginname}

{pluginname} is an AI-powered writing assistant that integrates directly into {productname}, providing instant text rewriting, summarization, correction, and contextual chat help based on organizational style guides. It includes automated review tools and enterprise-ready functionality that integrates with existing systems without requiring custom infrastructure.

Teams can implement a full suite of AI writing tools quickly, delivering efficient content workflows that maintain brand consistency and integrate smoothly with document management systems.

The core components of {pluginname} are:

* **xref:introduction-to-tinymce.adoc[{productname}]**: A modern rich text editor with dozens of features that improve writing workflows, including collaboration.

> * **xref:tinymceai-api-overview.adoc[AI service]**: A state-of-the-art back-end AI engine that incorporates multiple models and delivers high-quality content. *The hosted AI service is currently available in Cloud setup only.*

Reviewer: might be better to flip this, so we're talking about both from the start
```suggestion
{pluginname} is an AI-powered writing assistant platform with two core components:

* TinyMCE AI plugin**: Integrates the AI assistant directly into {productname}, providing instant text rewriting, summarization, correction, and contextual chat help based on organizational style guides. It includes automated review tools and enterprise-ready functionality that integrates with existing systems without requiring custom infrastructure. 
* **xref:tinymceai-api-overview.adoc[TinyMCE AI API]**: A state-of-the-art back-end AI engine that incorporates multiple models and delivers high-quality content. Any functionality available in the plugin is also available in the API, plus more, and the API can be used anywhere in your application -- not just in TinyMCE. *The hosted AI service is currently available in Cloud setup only.*

These components enable teams to implement a full suite of AI writing tools quickly, delivering efficient content workflows that maintain brand consistency and integrate smoothly with document management systems. In addition, the plugin and API use the same authorisation mechanisms and share conversation history, enabling ease of integration with your application's broader content authoring workflow -- not just within TinyMCE.

Both components support multiple AI models, enabling a unified UI and API across the models your application needs.
```

Reviewer: note: wasn't sure how to do the link on the first dot point, and the last line is kind of me shoving in a marketing point - feel free to delete if you want. probably needs a link to the available models section too I guess...

Author: Updated in [this commit](https://github.com/tinymce/tinymce-docs/pull/3976/commits/a87a4566c51628166c1b9b76730b76598f3dbb02) and included links as requested.

include::partial$misc/tinymceai-on-premises-early-access-note.adoc[]

[[features]]
== {pluginname} features

There are three main features of {pluginname}.

* xref:tinymceai-chat.adoc[**Chat**]: Interactive AI chats with history and persistent context.
* xref:tinymceai-review-plugin.adoc[**Review**]: Content analysis and proofreading, optimized for larger content.
* xref:tinymceai-actions-plugin.adoc[**Actions**]: Fast, stateless operations for specific tasks.

[NOTE]
Existing customers on paid plans can request a full featured free trial through the link:{accountpageurl}[Customer Portal].

[[integration-options]]
== Integration Options

> {pluginname} can be integrated with applications through two methods: the plugin (for the editor UI) or the AI service API (for custom integrations). See xref:tinymceai-integration-options.adoc[Integration Options] for details.

Reviewer: ```suggestion
{pluginname} can be integrated with applications through two methods: the plugin (for out-of-the-box editor features) or the AI service API (for custom integrations). See xref:tinymceai-integration-options.adoc[Integration Options] for details.
```
trying to make it sound less like plugin = editor and api = other things, since you could use the api in the editor with custom UI if you want

[[permissions]]
== Permissions

Developers can control access to AI features, models, and capabilities based on user roles, subscription tiers, and organizational requirements. Learn more about the xref:tinymceai-permissions.adoc[permissions system].

[[privacy-and-data-handling]]
== Privacy and data handling

[[regional-data-storage]]
=== Regional Data Storage

All data stored by {pluginname} follows the region settings of the {productname} Cloud Services environment, ensuring compliance with data residency requirements and optimal performance for the geographic location.

> NOTE: Data sent to LLM providers for processing is currently handled in the US region, regardless of the Cloud Services environment region. {productname} does not offer separate LLM processing regions to choose from.

Reviewer: I feel like these two paragraphs somewhat contradict each other? if we don't need to be vague, I think we should just say we use US region.
```suggestion
All data stored by Tiny for {pluginname} is handled in the US region. 

Data sent to LLM providers for processing is currently handled in the US region. {productname} does not offer separate LLM processing regions to choose from.
```
I don't know what variable you use for Tiny sorry. also, has Compliance check this wording? should we ask them to?
we'll need to update this when we release on-prem too, but that's a later problem

[[data-retention-policy]]
=== Data Retention Policy

Conversation data is automatically deleted after 12 months of inactivity, including:

* all conversation messages and history,
* attached documents, files, and web resources,
* conversation metadata and settings.

[[security]]
=== Security

All data is encrypted in transit and at rest with end-to-end encryption. Conversations and attachments are stored in secure cloud infrastructure with fine-grained access control and comprehensive permission systems.

[[on-premises-deployment]]
== On-Premises Deployment

On-premises deployment allows the AI service to run within an organization's infrastructure, giving teams more control over data and infrastructure.

> include::partial$misc/tinymceai-on-premises-early-access-note.adoc[]

Reviewer 2: After reading through this page, I don't know where to go next. I think it needs a `next step` section

Author: Updated in https://github.com/tinymce/tinymce-docs/pull/3976/commits/2b53be68e9870b9c7da999cfcc13c5012f79f845 to include expected next step links.
